### PR TITLE
Add a TRLN homepage with new additions widgets & a submasthead to rei…

### PR DESCRIPTION
**Pull request by seanaery**
…nforce TRLN vs. DUL scope. Closes CAT2-48 & CAT2-49.

### CODE REVIEW

File: app/views/catalog/_home_text.html.erb [hunk 1]
```diff
@@ -5,18 +5,21 @@
 
 <%= render partial: 'home_new_additions', locals: {
   type: 'duke_authors',
+  scope: 'catalog',
```
Comment (corylown) [inline app/views/catalog/_home_text.html.erb]: There are URL helpers that should set the right URL for various application functions based on context: `search_action_url(add_facet_params(field, value))`

File: app/views/trln/_submast.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<div id="trln-banner">
+  <div class="container">
+    <%= link_to t('blacklight.home.trln_submast_html').html_safe,'/trln', class: 'trln-home-link' %>
```
Comment (corylown) [inline app/views/trln/_submast.html.erb]: `search_action_url()` should generate the correct URL to /catalog or /trln depending on your current controller context.

File: app/views/catalog/_home_text.html.erb (hunk 1 above)
Comment (seanaery) [reply app/views/catalog/_home_text.html.erb]: @corylown -- Excellent, thanks!

File: app/views/trln/_submast.html.erb (hunk 2 above)
Comment (seanaery) [reply app/views/trln/_submast.html.erb]: Noted. Though in this particular case I always want the link in the TRLN banner to go to /trln.

File: app/views/catalog/_home_text.html.erb (hunk 1 above)
Comment (seanaery) [reply app/views/catalog/_home_text.html.erb]: @corylown I've refactored the scope to use `search_action_url` in a new commit -- just pushed up. Most of the logic lives in JS (in `new_additions.js`), so it's a little tricky, but I've revised how that JS assembles URLs for the AJAX GETs to catalog.json vs. trln.json endpoints and also the item URLs.

Comment (seanaery) [reply app/views/catalog/_home_text.html.erb]: Those changes are here: 
https://github.com/duke-libraries/dul-argon-skin/pull/177/commits/298a77d2090349a2a93d62f574c3597d6e5cbea0

File: app/views/trln/_submast.html.erb (hunk 2 above)
Comment (corylown) [reply app/views/trln/_submast.html.erb]: I still think it's better not to hard code paths inside the application. The helper will always return the right path to the route even if the path name changes.

Comment (seanaery) [reply app/views/trln/_submast.html.erb]: @corylown Good point -- will revise the latest commit w/this change.

Comment (corylown) [reply app/views/trln/_submast.html.erb]: thanks!

**Pull request merged**